ACPs/75-acceptance-proofs/README.md

```diff
@@ -49,60 +49,23 @@ Figure 4: The Validator accepts the P-Chain blocks and is now able to verify `Z`
 
 Note: The following is pseudocode.
 
-### ProposerVM
-
-The block header will include a proof-of-acceptance of the parent block which is
-a [Warp signature](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/README.md#awm-serialization) of the parent block id.
-
-```diff
-type BlockHeader struct {
-    PChainHeight    uint64
-    Timestamp       time.Time
-    Proposer        ids.NodeID
-+   AcceptanceProof warp.BitSetSignature
-  }
-```
-
 ### P2P
 
-#### Bootstrap
```

Comment:
I feel like this can be decoupled from this ACP (supporting bootstrapping using proofs)